connect/runtime/src/test/java/org/apache/kafka/connect/runtime/rest/RestServerTest.java

```diff
@@ -63,7 +63,7 @@
 import static org.junit.Assert.assertTrue;
 
 @RunWith(PowerMockRunner.class)
-@PowerMockIgnore({"javax.net.ssl.*", "javax.security.*", "javax.crypto.*"})
+@PowerMockIgnore({"javax.net.ssl.*", "javax.security.*", "javax.crypto.*", "javax.management.*"})
```

Comment:
Why was this necessary?

Reply:
It is related to a issue between log4j2 2.13.x and powermock; in short, it causes `java.lang.LinkageError` as of present.

Please see:

- https://stackoverflow.com/questions/41609436/powermock-after-log4j2-3-upgrade-could-not-reconfigure-jmx-java-lang-linkageerro
- https://stackoverflow.com/questions/16520699/mockito-powermock-linkageerror-while-mocking-system-class